src/adaptermanager.js

```diff
@@ -246,7 +246,9 @@ exports.enableAnalytics = function (config) {
 };
 
 exports.setBidderSequence = function (order) {
-  _bidderSequence = order;
+  if (order === CONSTANTS.ORDER.RANDOM) {
```

Comment:
Right, intention was to consolidate the `setBidderSequence` behavior into adaptermanager rather than having it spread across two functions. Noticed this while looking at this function when adding the `bidderSequence` config. Open to dropping this change if it's too out of scope